lib/travis/api/v3/services/beta_migration_request/create.rb

```diff
@@ -0,0 +1,37 @@
+module Travis::API::V3
+  class Services::BetaMigrationRequest::Create < Service
+    params :organizations
+    result_type :beta_migration_request
+
+    def run!
+      current_user = check_login_and_find(:user)
+      not_found(false, :beta_migration_request) unless current_user
```

Comment:
`check_login_and_find` will throw an exception, so I think this could be removed

Reply:
ok, fixed